payment/controllers.go

```diff
@@ -327,6 +328,48 @@ func GetOrderCreds(service *Service) handlers.AppHandler {
 	})
 }
 
+// GetOrderCredsByID is the handler for fetching order credentials by an item id
+func GetOrderCredsByID(service *Service) handlers.AppHandler {
+	return handlers.AppHandler(func(w http.ResponseWriter, r *http.Request) *handlers.AppError {
+		orderID := chi.URLParam(r, "orderID")
+		itemID := chi.URLParam(r, "itemID")
+		if orderID == "" || !govalidator.IsUUIDv4(orderID) {
+			return handlers.ValidationError("Error validating request url parameter",
+				map[string]interface{}{
+					"orderID": "orderID must be a uuidv4",
+				})
+		}
+		if itemID == "" || !govalidator.IsUUIDv4(itemID) {
+			return handlers.ValidationError("Error validating request url parameter",
+				map[string]interface{}{
+					"itemID": "itemID must be a uuidv4",
+				})
+		}
+
+		id := uuid.Must(uuid.FromString(orderID))
+		orderItemID := uuid.Must(uuid.FromString(itemID))
+
+		creds, err := service.datastore.GetOrderCredsByItemID(id, orderItemID)
+		if err != nil {
+			return handlers.WrapError(err, "Error getting claim", http.StatusBadRequest)
+		}
+
+		if creds == nil {
+			return &handlers.AppError{
+				Message: "Credentials do not exist",
+				Code:    http.StatusNotFound,
+				Data:    map[string]interface{}{},
+			}
+		}
+
+		w.WriteHeader(http.StatusOK)
+		if err := json.NewEncoder(w).Encode(creds); err != nil {
```

Comment:
Would be cool to use that sweet RenderContent function from #285

Reply:
Agreed! Copy/pasted https://github.com/brave-intl/bat-go/pull/302/commits/955f38a4bcd2e62e85c063d69ee54a46f4acefdf